src/form/Form.tsx

```diff
@@ -149,6 +149,7 @@ function Form({
 }: Props) {
   const formKey = _formName ?? _formKey;
   const [client, setClient] = useState(null);
+  const [authId, setAuthId] = useState<typeof initState.authId>();
```

Comment:
I need to create state because initState.authId is always undefined in the `getNewStep` while loop. Need to force re-evaluation of the rules once authId has been set